File: app/src/main/java/com/tukholko/assistant/app/Shop.java
package com.tukholko.assistant.app;

import com.google.gson.annotations.SerializedName;

public class Shop {
    @SerializedName("Local_shop_ID")
    public Integer localShopID;
    @SerializedName("Local_shop_name")
    public String localShopName;
    @SerializedName("Latitude")
    public Double latitude;
    @SerializedName("Longitude")
    public Double longitude;
}

File: app/src/main/java/com/tukholko/assistant/model/Shop.kt
package com.tukholko.assistant.model

import com.google.gson.annotations.SerializedName

data class Shop(
        @SerializedName("Local_shop_ID")
        var localShopID: Int,
        @SerializedName("Local_shop_name")
        var localShopName: String,
        @SerializedName("Country")
        var country: String,
        @SerializedName("City")
        var city: String,
        @SerializedName("Address")
        var address: String,
        @SerializedName("Latitude")
        var latitude: Double,
        @SerializedName("Longitude")
        var longitude: Double
)
